stmhal/modmachine.c

```diff
@@ -310,6 +314,19 @@ STATIC mp_obj_t machine_freq(mp_uint_t n_args, const mp_obj_t *args) {
         // re-init TIM3 for USB CDC rate
         timer_tim3_init();
 
+        // qqqqqqqq pppppppp nnnnnnnn nnmmmmmm
+        // qqqqQQQQ ppppppPP nNNNNNNN NNMMMMMM
+        // 222111HH HHQQQQPP nNNNNNNN NNMMMMMM
+        p = (p/2)-1;
+#if defined(MCU_SERIES_F7)
+#define FREQ_BKP BKP31R
+#else
+#define FREQ_BKP BKP19R
+#endif
+        RTC->FREQ_BKP = m | (n << 6) | (p << 16) | (q << 18)
+                    | (h << 22)
+                    | (b1 << 26)
+                    | (b2 << 29);      // TSP_20151030
```

Comment:
I assume TSP is your initials.  With git you don't need to make such comments because we can easily track who made what change and when.